main.py

```
@@ -0,0 +1,9 @@
+
+def factorial(num: int) -> int:
+    count = 1
```

Add a conditions that checks for values less zero. Should throw an error if the value is less than 0.